System/EnemyArchetypes.ini

```diff
@@ -2160,7 +2160,7 @@ BackupWeapon=(Weapon="SwatEquipment.Glock9mmHG",Chance=20)
 BackupWeapon=(Weapon="SwatEquipment.Glock19HG",Chance=20)
 BackupWeapon=(Weapon="SwatEquipment.BadguyColtM1911HG",Chance=20)
 BackupWeapon=(Weapon="",Chance=20)
-BackupWeapon=(Weapon="BrowningHPHG",Chance=20)
+BackupWeapon=(Weapon="SwatEquipment.BrowningHPHG",Chance=20)
 Equipment1=(Equipment="SwatSimpleEquipment.Glasses_a",Chance=30)
 Equipment1=(Equipment="SwatSimpleEquipment.Glasses_c",Chance=30)
 Equipment1=(Equipment="",Chance=40)
```

Comment:
what about this? here is no weapon at all

Reply:
Meaning there is a possibility for no backup weapon.